src/components/link-detail/styles.css

```diff
@@ -302,3 +302,8 @@
   display: inline-block;
   color: #de3535;
 }
+
+.link-detail-last-sync-time {
```

Comment:
Would you be able to make the font a bit bigger, maybe 18 or 24px? Also, could you use flexbox to right align the text? I suspect that will help visually link the text to the "resync" button.

Reply:
18px now, aligned right